test/test_parser.py

```diff
@@ -1,14 +1,13 @@
 import unittest
-
-import pyglet
+import os
 
 import pywavefront.parser
 
 class TestParsers(unittest.TestCase):
     def setUp(self):
-        pyglet.resource.path.append('@' + __name__)
-        pyglet.resource.reindex()
-        meshes = pywavefront.Wavefront('simple.obj')
+        # Append current path to locate files
+        folder = os.path.dirname(__file__) + '/'
```

Comment:
Please convert these to use `os.join`. Won't someone please think of the Windows users? 😄

Reply:
yeah... sorry about that Windows people :sweat_smile: